libpod/kube.go

```diff
@@ -220,12 +220,12 @@ func containerToV1Container(c *Container) (v1.Container, error) {
 			return kubeContainer, err
 		}
 		kubeContainer.VolumeMounts = volumes
-		return kubeContainer, errors.Wrapf(ErrNotImplemented, "volume names")
+		//return kubeContainer, errors.Wrapf(ErrNotImplemented, "volume names")
```

Comment:
If we are ignoring the error then remove this line, otherwise why are we commenting it out?